src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
Please, pay attention to access modifiers - leaving default is not a good practice